dataflow/flex-templates/streaming_beam/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM gcr.io/dataflow-templates-base/python3-template-launcher-base
```

Comment:
Can you add the copyright and license to all files? (except README and requirements.txt)

Reply:
Example Copyright / License:

```
# Copyright 2020 Google LLC
#
# Licensed under the Apache License, Version 2.0 (the "License");
# you may not use this file except in compliance with the License.
# You may obtain a copy of the License at
#
#    http://www.apache.org/licenses/LICENSE-2.0
#
# Unless required by applicable law or agreed to in writing, software
# distributed under the License is distributed on an "AS IS" BASIS,
# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
# See the License for the specific language governing permissions and
# limitations under the License.
```

I believe this should work both for the dockerfile and for any new Python sources.

Reply:
Thanks. Added this to Dockerfile.